protos/processor.proto

```diff
@@ -26,7 +26,21 @@ import "transaction.proto";
 //
 // The protocol_version field is used to check if the validator supports
 // requested features by a transaction processor.
+// Following are the versions supported:
```

Comment:
This comment says the rfc is at text/0000-raw-txn-header.md but it actually been moved to text/0023-raw-txn-header.md Please either update this or remove that line and just point to the pr